src/views/AdminSettings.vue

```diff
@@ -69,33 +69,4 @@ export default {
 </script>
 
 <style lang="scss" scoped>
-::v-deep {
-	input {
-		width: 300px;
-		vertical-align: middle;
-	}
-
-	select {
-		vertical-align: middle;
-	}
-
-	.icon-delete,
-	.icon-checkmark-color,
-	.icon-category-monitoring,
-	.icon-checkmark,
-	.icon-error {
-		display: inline-block;
-		width: 44px;
-		height: 44px;
-		vertical-align: middle;
-	}
-
-	.icon-checkmark-color.hidden {
-		display: none;
-	}
-
-	.error {
-		border-color: var(--color-error);
-	}
```

Comment:
Were those files also redundant? 😀

Reply:
icons from material-design-icons library either wrapped in the NcButton and already have these styles or have different classes, like `check-icon` instead of `icon-checkmark`
In both cases, they're not usable anymore

Reply:
same for error handling, we need to use `&.error :deep(.input-field__input)` to adjust input appearance